### ISSUE TITLE
Stepmania Crashes on startup:  Unhandled Microsoft C++ Exception

### ISSUE THREAD

**Issue by Neverlink** [stepmania/stepmania#1467]
After slamming a good 30-ish songs in my stepmania folder. Tries to start the game -> crash. Intuitively I deleted all folders/items containing strange symbols, but nothing changed. Reinstalling (quite a few times already) changes nothing as well. I've also tried installing the game in a different location, nothing changed.

Crash info: https://pastebin.com/Z1Y1zvSq
Log: https://pastebin.com/qDHMnssH

**Comment by SpoOkyMagician** [comment]
From the logs, it sounds like a graphics card thing. Is your graphics card driver up to date?

**Comment by Neverlink** [comment]
Yes.

**Comment by SpoOkyMagician** [comment]
Reminds me of a similar issue I had. Although, I was a bit annoyed at the time so I closed it... Perhaps there is something actually here that is a problem...

**Comment by SpoOkyMagician** [comment]
So, I found out something for my case. I am running in a local user. I added "run as admin" to the program and it worked fine. @Neverlink are you running a local user as well or admin?

**Comment by Neverlink** [comment]
Yes i've tried that, no difference. I am the only admin of the computer as well.

**Comment by SpoOkyMagician** [comment]
Darn... That is the only idea I have sorry. :(

**Comment by SpoOkyMagician** [comment]
I do have one other question that might be the issue. In your preferences, do you have autogen on? If so, turn it off and try again.

**Comment by IchiroYakuza** [comment]
> Yes i've tried that, no difference. I am the only admin of the computer as well.

I may be have the same problem as you. I hope this will help you to solve it!!

Right-click on the Stepmania and open file location - Stepmania 5 location folders, then delete "Cache" 'WARNING! You may lose score of yours, be sure to save if you want to!'
 
It should be like this if you not sure.   C:\Games\StepMania 5 <file directory
